sdk/spring/azure-spring-data-cosmos/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 ## Release History
 
-### 3.39.0-beta.1 (Unreleased)
-
-#### Features Added
-
-#### Breaking Changes
-
-#### Bugs Fixed
+### 3.39.0 (2023-10-07)
 
 #### Other Changes
+* Updated `azure-cosmos` to version `4.50.0`.
+* Updated `Spring Boot` to version `2.7.16`.
```

Comment:
We can remove this Spring Boot entry, since the Spring Boot is on top of the Spring Data layer, so the Spring Data Cosmos has no dependency on the Spring Boot.